Document CLPDECODE. (#236)
pinot-contrib/pinot-docs: basics/data-import/clp.md

## Searching and decoding CLP-encoded fields

> To decode CLP-encoded fields, users can use [CLPDECODE](../../configuration-reference/functions/clpdecode.md).

Reviewer: ```suggestion
To decode CLP-encoded fields, use [CLPDECODE](../../configuration-reference/functions/clpdecode.md).
```

> To search CLP-encoded fields, users can combine `CLPDECODE` with `LIKE`, however this may be expensive if there are a lot of rows to query.

Reviewer: ```suggestion
To search CLP-encoded fields, you can combine `CLPDECODE` with `LIKE`. Note, this may decrease performance when querying lot of rows.
```
